llpc/translator/lib/SPIRV/libSPIRV/SPIRVModule.cpp

```diff
@@ -633,11 +634,19 @@ bool SPIRVModuleImpl::importBuiltinSet(const std::string &BuiltinSetName,
   return true;
 }
 
+bool SPIRVModuleImpl::isNonSemanticInfoInstSet(const std::string &setName) const {
+  return (strstr(setName.c_str(), "NonSemantic.") == setName.c_str());
```

Comment:
I don't get this code. Can't `string::find` be used instead?
`setName.find("NonSemantic.") == 0`?
Or even `StringRef::startswith`, if llvm can be used in this part of the codebase.

Reply:
I agree. Using StringRef would make much more sense. See
https://llvm.org/docs/ProgrammersManual.html#llvm-adt-stringref-h
Probably best to pass StringRef in to the function.

Reply:
Thanks. Using StringRef now.